manifest.js

```diff
@@ -79,5 +79,10 @@ module.exports = {
 	remainingArticlesTest: {
 		path: 'bottom/remaining-articles',
 		lazy: true
-	}
+	},
+	tryFullAccess: {
+		path: 'top/lazy',
+		lazy: true,
+		guruQueryString: 'offerId=683995bf-ed3f-2fd6-d589-1628ebad1629',
```

Comment:
this needs to be the production offer ID:
```suggestion
		guruQueryString: 'offerId=41218b9e-c8ae-c934-43ad-71b13fcb4465',
```